Shuttlerock/actions
# [STUDIO-535] Moving an epic task to `In Development` should create an Epic PR

## Moving an epic task to &#x60;In Development&#x60; should create an Epic PR

[Jira Tech task](https://shuttlerock.atlassian.net/browse/STUDIO-535)



## How to test the PR

Triggering the job for issue `236` should create 2 pull requests - the epic PR and the regular PR (which should be based on the epic branch).

```
$ act --job trigger_action --eventpath src/actions/trigger-action/__tests__/fixtures/createPullRequestForJiraIssue.ts.json
```

## Deployment Notes

None.
